site/content/docs/main/release-instructions.md

```diff
@@ -7,7 +7,7 @@ This page covers the steps to perform when releasing a new version of Velero.
 
 ## General notes
 - Please read the documented variables in each script to understand what they are for and how to properly format their values.
-- You will need to have a remote named `upstream` pointing to the Velero repository so the scripts can work properly.
+- You will need to have an upstream remote configured to use to the [vmware-tanzu/velero](https://github.com/vmware-tanzu/velero) repository. You can check this using `git remote -v`.
```

Comment:
Should we mention the default of `upstream`?

Reply:
Mostly looking for feedback from others here - it's already documented in the script, so it may be fine being omitted here.

Reply:
Yep, I can update that. I don't think it hurts to mention the default value again here.